### ISSUE TITLE
Choose content license for Monash modules

### ISSUE THREAD

**Issue by cynthiahqy** [numbats/monash-learnr-modules#11]
@robjhyndman thoughts on licensing? -- maybe https://chooser-beta.creativecommons.org

**Comment by robjhyndman** [comment]
How about [CC BY-NC-SA 4.0](https://creativecommons.org/licenses/by-nc-sa/4.0/?ref=chooser-v1)?

**Comment by cynthiahqy** [comment]
Looping @mitchelloharawild into this -- 

No commercial might need more thought @robjhyndman 

- likely precludes usage in paid workshops? If that is what we want. 
- for comparison, https://mine-cr.com/project/datasciencebox/ uses https://creativecommons.org/licenses/by-sa/4.0/

**Comment by mitchelloharawild** [comment]
I was thinking of CC BY-SA 4.0 as the default / recommended for module contributions, with some flexibility for maintainers to choose their preferred license up to some compatibility limit on restrictive modules.

I think NC would also be within the acceptable set of licenses - although we would need to treat it more carefully in the learnr.academy organisation to ensure that it isn't used anywhere for funding the learnr org (e.g. in preventing paid course websites remixing using CC BY-NC-SA modules).

**Comment by robjhyndman** [comment]
Fair enough. I was thinking of stopping a company like DataCamp from adapting our materials and then making money from it. I don't think NC would preclude paid workshops as then you are paying for the workshop presenter time, not necessarily for the materials. 

But CC BC-SA would be fine. It actually inhibits most commercial use in any case because the free version will be available, so why pay?
